promptsource/templates/qasc/templates.yaml

```
@@ -1,82 +1,119 @@
 dataset: qasc
 templates:
   3e1e6ca0-b95e-4e68-bb6a-cd47c8429658: !Template
-    answer_choices: null
+    answer_choices:
+    - 'Yes'
+    - 'No'
     answer_choices_key: null
     id: 3e1e6ca0-b95e-4e68-bb6a-cd47c8429658
     jinja: "If I tell you that {{combinedfact[0]|capitalize}}{{ combinedfact[1:]|trim('.')\
       \ }}, and ask you the question \"{{ question[0]|lower }}{{ question[1:] }}\"\
       , is the correct answer \"{{ choices.text[0][0]|lower}}{{ choices.text[0][1:]|trim('.')\
-      \ }}\"? ||| \n{% if answerKey == choices.label[0] %} Yes {% else %} No {% endif\
-      \ %}"
+      \ }}\"? \n\n||| \n\n{% if answerKey == choices.label[0] %} Yes {% else %} No\
+      \ {% endif %}"
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
       choices_in_prompt: false
-      metrics: []
-      original_task: null
-    name: is-correct-0
+      metrics:
+      - Accuracy
+      original_task: false
+    name: is-correct-1
     reference: ''
   40ef67db-dff3-4e7b-b167-f7e54b400c74: !Template
     answer_choices: null
-    answer_choices_key: null
+    answer_choices_key: choices ; text
     id: 40ef67db-dff3-4e7b-b167-f7e54b400c74
     jinja: "{{ fact1[0]|capitalize }}{{ fact1[1:]|trim|trim('.') }}, and {{fact2[0]|lower\
-      \ }}{{ fact2[1:]|trim|trim('.') }}. Given this, {{ question[0]|lower }}{{question[1:]|trim('?')\
-      \ }} between the following options: {% for option in choices.text %}\"{{ option|trim|trim('.')}}\"\
-      {% if loop.index == choices.text|length\n      -1 %} or {% elif  loop.index\
-      \ < choices.text|length -1 %}, {% endif %}{% endfor %}? ||| {% for choice in\
-      \ choices.label %} {% if choice == answerKey %}{{ choices.text[loop.index -\
-      \ 1][0]|capitalize }}{{ choices.text[loop.index - 1][1:] }}{% endif %}{% endfor\
-      \ %}   "
+      \ }}{{ fact2[1:]|trim|trim('.') }}. Given these facts, {{ question[0]|lower\
+      \ }}{{question[1:]|trim('?') }} between the following options: {{ choices.text\
```

Grammatical nit: Use "between" when they are only 2 options. Use "among" for more than 2 options.
```suggestion
      \ }}{{question[1:]|trim('?') }} among the following options: {{ choices.text\
```